src/assets/scss/application.scss

```diff
@@ -101,3 +101,11 @@ $govuk-assets-path: "/public/";
 #submitDetails {
   @include govuk-responsive-margin(4, "top");
 }
+
+.couldNotMatchPrompt {
+  color: govuk-colour("black");
+  font-size: 22px;
```

Comment:
Setting the font to a set px can cause issues across different screen sizes, is there a reason that we're not using the [government design system typography](https://design-system.service.gov.uk/styles/typography/)?